### PR TITLE
chore: minor fixes to the review of essentials

### PR DESCRIPTION
**Pull request by tomaszantas**
Description
---

1. Position of the text in the text input - added line height as it moves the text bit lower

https://user-images.githubusercontent.com/11715931/166669121-e9714cba-0bf9-4f8b-93c4-ae7b58dba403.mov

2. Switched styled label to the Text component so it uses the Avenir font

image

3. Change color of texts in Balance

image

4. Fix loader changing size of Tabs

https://user-images.githubusercontent.com/11715931/166674570-d70bc280-b024-48fc-b6fa-d8fd64d49684.mov

Motivation and Context
---

Review of essential issues:

#16 
#123 

How Has This Been Tested?
---

See screens and recordings attached above.

### CODE REVIEW

File: applications/launchpad/gui-react/src/components/Text/types.ts [hunk 1]
```diff
@@ -37,8 +37,9 @@ export interface TextProps {
     | 'h6'
     | 'p'
     | 'span'
+    | 'label'
     | AnimatedComponent<
-        'h1' | 'h2' | 'h3' | 'h4' | 'h4' | 'h5' | 'h6' | 'p' | 'span'
+        'h1' | 'h2' | 'h3' | 'h4' | 'h4' | 'h5' | 'h6' | 'p' | 'span' | 'label'
```
Comment (tarnas14) [inline applications/launchpad/gui-react/src/components/Text/types.ts:42]: update jsdoc

**Review by tarnas14 — APPROVED** [review]

**Comment by tarnas14** [comment]
looks good, one comment about jsdoc :+1:

**Review by corquaid — APPROVED** [review]
LGTM!

**Comment by tomaszantas** [comment]
> looks good, one comment about jsdoc 👍

Going to add with the next PR

**Pull request merged**